groups/csa/csamisc/csamisc_emptystringinitialisation.cpp

```diff
@@ -0,0 +1,140 @@
+#include <clang/ASTMatchers/ASTMatchers.h>
+#include <clang/ASTMatchers/ASTMatchFinder.h>
+#include <clang/Lex/Lexer.h>
+
+#include <clang/AST/Stmt.h>
+#include <csabase_analyser.h>
+#include <csabase_debug.h>
+#include <csabase_diagnostic_builder.h>
+#include <csabase_registercheck.h>
+#include <csabase_report.h>
+#include <csabase_util.h>
+#include <csabase_visitor.h>
+#include <set>
+
+#include <iostream>
+
+using namespace csabase;
```

Comment:
Please remove this so that we can easy tell apart between clang and non-clang component